### ISSUE TITLE
some questions about the training code

### ISSUE THREAD

**Issue by l1346792580123** [pengHTYX/PSHuman#21]
Thanks for opening source such great work！I have some questions about the training code. There is a `from_pretrained_linear` function in the line [413](https://github.com/pengHTYX/PSHuman/blob/f2a635126dca3bf0526b4a08723cb84a79b57cab/train_mvdiffusion_unit_unclip.py#L413) of `train_mvdiffusion_unit_unclip.py`, while I do not find this function in the `DDPMScheduler` documentation. Is there any difference between this function and `from_pretrained`? And I have noticed that you have flipped the third and fourth images when [processing the training data](https://github.com/pengHTYX/PSHuman/blob/f2a635126dca3bf0526b4a08723cb84a79b57cab/mvdiffusion/data/dreamdata.py#L388) and [performing inference](https://github.com/pengHTYX/PSHuman/blob/f2a635126dca3bf0526b4a08723cb84a79b57cab/inference.py#L152). Does this perform better than not flipping?

**Comment by pengHTYX** [comment]
Hi, thanks for your interest. 

For the first question, we modify the 'beta_schedule' of DDIM scheduler in SD2.1 to 'linear' for high resolution finetuning. You can follow [the noise setting](https://github.com/pengHTYX/Era3D/blob/a2ce68da53c0dc4df403112c53692b9ba893a4f0/configs/train-unclip-512-6view.yaml#L91) and [line 370-372](https://github.com/pengHTYX/Era3D/blob/a2ce68da53c0dc4df403112c53692b9ba893a4f0/train_mvdiffusion_unit_unclip.py#L370) to fix this. I will fix this bug soon.

Regarding the flipping, we did not ablate it. However, intuitively, it could help better alignment for front-back view pairs.

**Comment by l1346792580123** [comment]
Thanks for your reply!
